admin/client/components/Forms/EditForm.js

```diff
@@ -98,6 +101,73 @@ var EditForm = React.createClass({
 			confirmationDialog: null,
 		});
 	},
+	updateItem () {
+		const { data, list } = this.props;
+		const editForm = this.refs.editForm.getDOMNode();
+		const formData = new FormData(editForm);
+
+		// Clear alerts
+		this.setState({
+			err: null,
+			success: null,
+			loading: true,
+		});
+
+		list.updateItem(data.id, formData, (err, data) => {
+			// TODO: implement smooth scolling
+			scrollTo(0, 0); // Scroll to top
+			if (err) {
+				this.setState({
+					err: err,
+					loading: false,
+				});
+			} else {
+				// Success, display success flash messages, replace values
+				// TODO: Update key value
```

Comment:
Right now it just jumps you the top on update and it's a bit jarring. Figured it might be nicer with an animated scroll so the user has a better idea of what's going on. When we get to inline errors on the fields it'd be nice to scroll to the top field with an error instead of the top. Not super important though so I just left a comment to revisit it later once everything else is done.